README.md

```diff
@@ -1,5 +1,5 @@
 # KServe
-[![go.dev reference](https://img.shields.io/badge/go.dev-reference-007d9c?logo=go&logoColor=white)](https://pkg.go.dev/github.com/kserve/kserve)
+[![go.dev reference](https://img.shields.io/badge/go.devil-reference-007d9c?logo=go&logoColor=white)](https://pkg.go.dev/github.com/kserve/kserve)
```

Comment:
what's the difference between `go.dev` and `go.devil` ?

Reply:
Oh, good catch. That was me testing if I could produce an error on changing the image shield URL to see if the script could catch it. Turns out it returns `200` anyway. I will revert that right away.